mlrun/common/model_monitoring/__init__.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+__all__ = [
+    "EventFieldType",
+    "EventLiveStats",
+    "EventKeyMetrics",
+    "TimeSeriesTarget",
+    "ModelEndpointTarget",
+    "FileTargetKind",
+    "ProjectSecretKeys",
+    "ModelMonitoringStoreKinds",
+    "EndpointType",
+    "ModelMonitoringMode",
+    "create_model_endpoint_uid",
+]
```

Comment:
why?
using all simply tells the importer "Hi, just is what you can import" but basically you added everything you import anyway. usually you want to use it when you want to _limit_ what the developers can import (or see when they import)

Reply:
Got it, thanks. It is indeed unnecessary to use `__all__` here